lib/ui/sign_transaction/failed/sign_transaction_failed_page.dart

```diff
@@ -47,27 +50,38 @@ class SignTransactionFailedPage extends StatelessWidget {
         body: Stack(
           children: [
             const HyphaHalfBackground(backgroundColor: HyphaColors.error, showTopBar: false),
-            Column(
-              children: [
-                const SizedBox(height: 80),
-                Image.asset('assets/images/warning.png', width: 240, height: 240),
-                Padding(
-                  padding: const EdgeInsets.only(left: 45, right: 45, top: 24),
-                  child: Text('Sorry...', textAlign: TextAlign.center, style: context.hyphaTextTheme.mediumTitles),
-                ),
-                const SizedBox(height: 16),
-                Padding(padding: const EdgeInsets.only(left: 45, right: 45), child: failedText),
-                const SizedBox(height: 16),
-                Padding(
-                  padding: const EdgeInsets.only(left: 45, right: 45),
-                  child: Text(
-                    'Please try again by triggering the transaction from the website or app. Sorry for the inconvenience.',
-                    style: context.hyphaTextTheme.ralMediumBody.copyWith(color: HyphaColors.midGrey),
-                    textAlign: TextAlign.center,
-                  ),
+            SingleChildScrollView(
+              child: Padding(
+                padding: const EdgeInsets.symmetric(horizontal: 45),
+                child: Column(
+                  children: [
+                    const SizedBox(height: 80),
+                    Image.asset('assets/images/warning.png', width: 240, height: 240),
+                    const SizedBox(height: 24),
+                    Text('Sorry...', textAlign: TextAlign.center, style: context.hyphaTextTheme.mediumTitles),
+                    const SizedBox(height: 16),
+                    failedText,
+                    const SizedBox(height: 16),
+                    Text(
+                      'Please try again by triggering the transaction from the website or app. Sorry for the inconvenience.',
+                      style: context.hyphaTextTheme.ralMediumBody.copyWith(color: HyphaColors.midGrey),
+                      textAlign: TextAlign.center,
+                    ),
+                    const SizedBox(height: 16),
+                    ExpansionTile(
+                      title: const Text('Tap to see full error'),
+                      collapsedTextColor: context.hyphaTextTheme.ralMediumBody.color,
+                      textColor: context.hyphaTextTheme.ralMediumBody.color,
+                      children: [
+                        Padding(
+                          padding: const EdgeInsets.only(bottom: 8),
+                          child: Text(error.message, style: context.hyphaTextTheme.ralMediumBody),
```

Comment:
not sure error.message is what we need? Does that contain the blockchain error message? Did we test this?